src/main/java/com/ays/backend/user/controller/payload/response/AuthResponse.java

```diff
@@ -3,9 +3,15 @@
 import lombok.Builder;
 import lombok.Data;
 
+import java.util.List;
+
 @Data
 @Builder
 public class AuthResponse {
     String message;
+    String username;
     String accessToken;
+    String refreshToken;
+    Long expireDate;
+    private List<String> roles;
```

Comment:
It might be healthier to use a Set instead of a List as it could provide faster search times.

Reply:
I tried to use Set instead of List according to your approach.